src/main/java/de/dataelementhub/model/service/NamespaceService.java

```diff
@@ -226,4 +235,20 @@ public void release(DSLContext ctx, int userId, String urn) {
           "Element Type is not supported: " + identification.getElementType());
     }
   }
+
+  /**
+   * Check for duplicate languages in definitions.
+   */
+  private boolean hasDuplicateLanguageDefinitions(Element element) {
```

Comment:
This is duplicating the code from ElementService. Please check if it is feasible to make the method in elementservice public (and static) and use it here as well.

Reply:
Since the method is about filtering definitions I placed it in the DefinitionHandler then imported it in the ElementService & NamespaceService. That should solve the problem with duplicated code.